spec/data-model/README.md

```diff
@@ -237,19 +234,16 @@ interface VariableRef {
 A `FunctionAnnotation` represents a _function_ _annotation_.
 The `name` does not include the `:` starting sigil.
 
-Each _option_ is represented by an `Option`.
+Each _option_ is represented by an `Options` key-value mapping.
```

Comment:
Document the contents of `Options` in its own place and document `Options` here.

```suggestion
`Options` is a map containing the _expression_'s _options_.
```

Reply:
```suggestion
`Options` is a key-value mapping containing options,
and is used to represent the _annotation_ and _markup_ _options_.
```